frontend/src/components/NoData/NoData.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Result } from 'antd';
+
+const NoData: React.FC = () => <Result status="error" title="No data to display"></Result>;
```

Comment:
auto closing? `/>`

Reply:
`const NoData: React.FC = () => <Result status="error" title="No data to display" />;`